Gems/CsvSpawner/gem.json

```diff
@@ -0,0 +1,30 @@
+{
+  "gem_name": "CsvSpawner",
+  "version": "1.0.0",
+  "display_name": "CsvSpawner",
+  "license": "Apache-2.0",
+  "license_url": "https://opensource.org/licenses/Apache-2.0",
+  "origin": "Robotec.ai",
+  "origin_url": "https://github.com/RobotecAI/robotec-o3de-tools",
+  "type": "Code",
+  "summary": "Component that spawns prefabs using coordinates stored in csv file. It supports XYZ format as well as WGS84 coordinate system.",
+  "canonical_tags": [
+    "Gem"
+  ],
+  "user_tags": [
+    "CsvSpawner"
+  ],
+  "platforms": [
+    ""
+  ],
+  "icon_path": "preview.png",
+  "requirements": "Requires ROS2 Gem",
+  "documentation_url": "",
+  "dependencies": [
+    "ROS2"
```

Comment:
Is any particular version of this Gem needed? Should it be tight to a certain version?